DSharpPlus.Rest/DiscordRestClient.cs

```diff
@@ -11,17 +11,10 @@
 
 namespace DSharpPlus;
 
+using Caching;
```

Comment:
nuh uh

Reply:
please for the sake of my sanity only submit reviews when i ask for them. This branch is highly work in progress